hs_app_timeseries/migrations/custom_data_migration_20160718.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from django.db import migrations
+
+from hs_core.hydroshare.utils import resource_modified
+from hs_app_timeseries.models import TimeSeriesResource
+
+
+def delete_extracted_metadata(apps, schema_editor):
+    # For all existing timeseries resources, delete all resource specific metadata.
+    # This way we can keep the resource GUID and allow the resource owners to manually
+    # delete the existing SQlite file and then add a new SQlite file to regenerate
+    # resource specific metadata
+    log = logging.getLogger()
+    for res in TimeSeriesResource.objects.all():
```

Comment:
immediately above this line, you need

    TimeSeriesResource = apps.get_model('hs_app_timeseries', 'TimeSeriesResource')